docs/content/en/docs/Coding Milestones/PoC2023q1/environments/dev-env.md

```diff
@@ -6,6 +6,479 @@ description: >-
+
+  - For example: create a kind cluster
+
+    ```bash
+    kind create cluster --name florin
+    ``` 
+    
+#### 2. Deploy the KCP-Edge platform:
+    
+  * Step-1: Clone this repo:
+
+    ```bash
+    git clone -b dev-env-v2 https://github.com/dumb0002/edge-mc.git
+    ```
+
+  * Step-2: change into the following directory path:
+
+    ```bash
+      cd edge-mc/environments/dev-env
+    ```
+
+  * Step-3: Deploy kcp-edge
+
+    ```bash
+    ./install_edge-mc.sh --stage 0
+    ```
+
+    This will start `kcp` and create/deploy the following components:
+
+    - 4 kcp workspaces: edge service provider workspace (`espw`), inventory management workspace (`imw`) and workload management workspace (`wmw`) under my-org workspace
+
+    ```bash
+    kubectl ws tree
+    .
+    └── root
+        ├── compute
+        ├── espw
+        ├── imw-1
+        └── my-org
+            └── wmw-1
+    ```
+    - 3 kcp-edge controllers: [edge-scheduler](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/edge-scheduler/), [mailbox-controller](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/mailbox-controller/) and [placement-translator](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/placement-translator/)
+
+    ```bash
+    ps aux | grep -e mailbox-controller -e placement-translator -e cmd/scheduler/main.go
+    user      2898   0.0  0.1 34922264  45188 s004  S     3:22PM   0:01.84 go run cmd/scheduler/main.go -v 2 --root-user shard-main-kcp-admin --root-cluster shard-main-root --sysadm-context shard-main-system:admin --sysadm-user shard-main-shard-admin
+    user      2872   0.0  0.2 34925136  56132 s004  S     3:22PM   0:02.44 go run ./cmd/mailbox-controller --inventory-context=shard-main-root -v=2
+    user      2929   0.0  0.2 34922964  69724 s004  S     3:22PM   0:03.74 go run ./cmd/placement-translator --allclusters-context shard-main-system:admin
+
+    ```
+
+#### 3. Connect your edge pcluster to the kcp-edge platform:
+
+  * Step-1: Populate the `imw`: enter the target workspace:
+
+    ```bash
+        kubectl ws root:imw-1
+    ```
+    
+  * Step-2: create a SyncTarget object to represent your edge pcluster. For example:
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: workload.kcp.io/v1alpha1
+    kind: SyncTarget
+    metadata:
+      name: sync-target-f
+      labels:
+        example: si
+        extended: non
+    spec:
+      cells:
+        foo: bar
+    EOF
+    ```
+
+  * Step-3: create a Location object describing your edge pcluster. For example:
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: scheduling.kcp.io/v1alpha1
+    kind: Location
+    metadata:
+      name: location-f
+      labels:
+        env: prod
+    spec:
+      resource: {group: workload.kcp.io, version: v1alpha1, resource: synctargets}
+      instanceSelector:
+        matchLabels: {"example":"si", "extended":"non"}
+    EOF
+    ```
+
+    A location and synctarget objects will be created:
```

Comment:
"will be" is future tense, but two of these three were created in the past at the time when the reader gets here.  I think this is really trying to say to delete the `default` Location if it is present.